index.js

```diff
@@ -165,6 +166,11 @@ module.exports = function (cmd, args, opts) {
 
 			err.stdout = stdout;
 			err.stderr = stderr;
+
+			if (!parsed.opts.reject) {
+				return err;
```

Comment:
Should we also add the following:

```js
err.failed = true;
```

And `failed: false` in the passing result?

Also, probably worth adding `code: 0` to the passing result?

If you are using this option, you are probably going to write conditionals around the result, and it would be nice to have as many consistent properties between the `pass` and `fail` results as possible.